.github/workflows/build.yml

```diff
@@ -186,9 +186,12 @@ jobs:
        fail-fast: false
        matrix:
          os: [windows-2019]
-         version: [jdk]
```

Comment:
Doesn't this still need "jdk" ? ie.jdk(head) "20"

Reply:
hmm yeah I'll add it back now

Reply:
@andrew-m-leonard updated PTAL